docs/UserGuide.md

```diff
@@ -546,6 +544,33 @@ as the assignment is not specified `assignment.json`
 Calling `addGrade` without any fields will show the list of assignments in `assignment.json`.
 </box>
 
+--------------------------------------------------------------------------------------------------------------------
+
+### Removing grades from a contact `removeGrade`
+
+Removes an assignment and its grades from a contact.
+
+<box type="definition" icon=":fa-solid-spell-check:" light>
+
+<md>**Format: `removeGrade n/NAME asgn/ASSIGNMENT_NAME`**</md>
+
+</box>
+
+<box type="definition" icon=":fa-solid-book:" light>
+
+<md>**Examples:**</md>
+
+Assuming John Doe has `Ex01` assignment with a score of `5`.
+
+`removeGrade n/John Doe asgn/Ex01` will remove the `Ex01` assignment from contact John Doe.
+
+Calling `removeGrade n/John Doe asgn/Ex01` again will throw an error since the assignment has already been removed.
+To add a new assignment, refer to `addGrade` command above.
```

Comment:
I think we can provide a hyperlink to the add grade command here

Reply:
Got it! Will add it in!